artifact/image/image_test.go

```diff
@@ -25,10 +26,11 @@ import (
 	depTypes "github.com/aquasecurity/go-dep-parser/pkg/types"
 )
 
-func TestArtifact_Inspect(t *testing.T) {
+func Test_imageArtifact_Inspect(t *testing.T) {
```

Comment:
changed to make it easier to trace hashes change, I forgot to change it back :facepalm:

Reply:
Will you change it back again? Some Go tools align the convention like TestStructName_MethodName. It allows us to jump to test code easily.

Reply:
yup will do